package.json

```diff
@@ -33,7 +33,13 @@
     "resolve-from": "^1.0.0"
   },
   "devDependencies": {
-    "ava": "0.0.3",
+    "ava": "*",
+    "xo": "^0.16.0",
     "heapdump": "^0.3.7"
+  },
+  "xo": {
+    "rules": {
+      "import/no-dynamic-require": "off"
```

Comment:
@SamVerschueren I added that there. Mostly because I had to turn it off both in the code and tests, so was better placed there.